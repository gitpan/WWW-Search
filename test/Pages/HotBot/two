http://www.hotbot.com/text/default.asp?SM=SC&MT=IG%2D88&DE=2&DC=100
http://www.hotbot.com/text/default.asp?act.next=next&MT=IG%2D88&SM=SC&DC=100&BT=T
