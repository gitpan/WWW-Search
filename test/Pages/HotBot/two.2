
<html>
<head><title>HotBot results: Zuckuss (201+)</title></head>
<body>
<br>
<form action="/text/default.asp">
<font face=Arial,Geneva,sans-serif>&nbsp;&nbsp;&nbsp;<b>HotBot: Text-only version</b> | 
<a href="/?act.revise=+Revise+&MT=Zuckuss&SM=SC&DC=100&base=200">Full graphics version</a>
<P>
</font>
<font face="Arial, Helvetica" size="2">
<p>
&nbsp;&nbsp;&nbsp;<B>look for:</b>&nbsp;<SELECT name="SM" >
<OPTION value="MC">all the words
<OPTION value="SC" selected>any of the words
<OPTION value="phrase">exact phrase
<OPTION value="title">the page title
<OPTION value="name">the person
<OPTION value="url">links to this URL
<OPTION value="B">Boolean phrase
</SELECT>
&nbsp;<INPUT type=text name="MT" value="Zuckuss" size=30 maxlength=300>
&nbsp;<input type="submit" name="search" value="Search"><br>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<input type="SUBMIT" name="act.new" value=" New ">&nbsp;<input type="SUBMIT" name="act.revise" value=" Revise ">&nbsp;<input type="SUBMIT" name="act.super" value=" More Options ">
<br>
</font>
<font face="Arial, Helvetica" font size="1">


 
</font>
<br>&nbsp;<!-- adSpaceCOM --><!-- AdSpace program=hotbotresults size=large SERVER_NAME=www.hotbot.com KW=zuckuss KW=zuckuss ALL_KW=zuckuss erlicat=nothing// section=erli/nothing// cobrand= REMOTE_HOST=copper.dulles.tasc.com RH1=com RH2=tasc.com RH3=dulles.tasc.com RH4=copper.dulles.tasc.com REMOTE_ADDR=147.81.3.197 R_UNIQID=3soJEdZpxGAx P_UNIQID=3soJEdZpxGQx hbf=adult hbf=language:any --><a href="http://nsads.hotwired.com/event.ng/Type=click&ProfileID=8593&RunID=15406&AdID=20451&GroupID=1&FamilyID=2780&TagValues=2.5.6.25.145.156.174.186.258.374.389.409.526.539.6456.6624.6820.6830.72561.75378.75606&amp;params.tag.ALL_KW=zuckuss&Redirect=http:%2F%2Fwww.cars.com%2F%3Fhotbotad%3Dca9906002" TARGET="_top"><img src="http://static.wired.com/advertising/blipverts/carscom/plane_468.gif" BORDER=1 height=60 width=468 alt="Enter the Cars.com Cars of Summer Sweepstakes"></a><!-- /AdSpace --><!-- /adSpaceCOM -->


<blockquote>
<b>Returned:&nbsp;160&nbsp;Matches&nbsp;
<br>Results for &quot;Zuckuss&quot;</b><br>


<br>
<font face="Verdana, Arial, Helvtica" size="1">
<br>
<br>

<!-- BRES --> 



<p>&nbsp;</p>


<b>Sorry, your search yielded no results.</B><Br>
Please <a href="/?act.revise=+Revise+&MT=Zuckuss&SM=SC&DC=100&base=200">revise</a> your search query and try again.<P>



<br>
<!-- uncomment to see debug info 
<pre><BR>----<br></pre-->
<p>&nbsp;</p>


<INPUT type=hidden name="DC" value="100" >
<INPUT type=hidden name="base" value="200" >

<!-- ERES -->

<INPUT type=hidden name="_v" value="2" >
<INPUT type=hidden name="OPs" value="MDRTP" >

</form>
<p>

<!-- hotbotMon:success -->
<br>
<a href="/addurl.asp">Add URL</a> | <a href="mailto:wow@wired.com">Send&nbsp;us&nbsp;feedback</a>&nbsp;| 
<a href="http://www.hotwired.com/jobs/">Work&nbsp;at&nbsp;HotBot</a>&nbsp;| <a href="http://www.wired.com/home/advertising/">Advertise&nbsp;on&nbsp;HotBot</a>&nbsp;| 
<a href="http://www.wired.com/home/">Company&nbsp;Information</a>&nbsp;| 
<a href="http://www.wired.com/home/digital/privacy/">Our&nbsp;Privacy&nbsp;Policy</a>&nbsp;|
<a href="/help/">Help</a>&nbsp;| 
<a href="/text/"><nobr>Text-only version</nobr></a> 
<p>
<nobr><a href="http://www.wired.com/home/copyright.html">Copyright</a> &copy; 1994-99 Wired Digital Inc. All rights reserved.<br></nobr>

</font>
</blockquote>
</body>
</html>

