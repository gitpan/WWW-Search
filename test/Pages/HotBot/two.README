This set of test-result pages was created on Tue Aug 17 09:44:23 1999
