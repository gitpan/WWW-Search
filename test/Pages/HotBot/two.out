http://www.netmonger.net/~ig88
http://www.geocities.com/Area51/Dunes/7395/
http://disc.server.com/Indices/38974.html
http://pages.prodigy.com/IG88/swhome1.htm
http://www.turbotek.net/gheryp/sw/decks/deckd23.html
http://members.aol.com/jorbloom/
http://www.lionking.org/forum/Forum5/HTML/001418.html
http://www.gerf.org/~nixon
http://www.a1.com/skysccg/dagobah.html
http://members.wbs.net/homepages/r/a/v/ravenjedimistress.html
http://www.telecom.csuhayward.edu/~aleung/gbounty.htm
http://members.xoom.com/chemosh66/3dimg4.html
http://anotheruniverse.com/toys/toyshelf/pwr_for2.html
http://www.theforce.net/humor/emporium
http://theforce.net/jedicouncil/surveys/survey20.shtml
http://www.jedicouncil.net/surveys/survey20.shtml
http://idgames.gameaholic.com/idgames/quake2/graphics/skins/m-o/ninja.txt
http://enterprise.hb.se/~nibrs
http://www.toyarchive.com/Collections.html
http://www.jeuxvideo.com/btajv99/etajvhtm/01101113.htm
http://www2.lawinfo.com/boards/paralegals/topic530/2.27.99.12.13.43.html
http://pages.map.com/starwars/figures.html
http://168.229.236.7/~cc/picts.html
http://www.bookbuyer.com/aisles/titles/672069.htm
http://sac.uky.edu/~wjbigg00/jbswccg.htm
http://www.nground.com/catsw.html
http://home.rhein-zeitung.de/~seveline/swccg.htm
http://www.toysrgus.com/images-kennercarded.html
http://www.dailysci-fi.com/news/634.html
http://stats.deathcon.com/quake2-gt/Outer_Hangar/Deaths.html
http://www.superior.net/~fubman/links1.html
http://www.praetor.com/gab/teams/emerald_team.htm
http://www.his.se/ida/~a94jeska/starwars.html
http://www.ida.his.se/ida/~a94jeska/starwars.html
http://www.angelfire.com/tx/dalefranklin
http://www.gis.net/~mcfadden/jmm/swwritea.html
http://www.16bit.com/aswn/toytalk.htm
http://bigsun.wbs.net/homepages/k/y/l/kylemckorey.html
http://cs.millersv.edu/~mrstenin/swlego.html
http://www.starwarscards.net/swf.htm
http://www.idggames.net/gamepro/features/articles/0,1306,4263,00.html
http://www.games.net/gamepro/features/articles/0,1306,4019,00.html
http://www.pcgamesmag.com/gamepro/features/articles/0,1306,4263,00.html
http://pcgamesmag.com/gamepro/features/articles/0,1306,4263,00.html
http://209.203.239.166/reviews/frank/star_wars_sote
http://comunidades.infosel.com/starwars/articulos/precuela/1075
http://gamepro.com/exclusives/ex_74.html
http://www.mania.com/toys/toynews/toytime080897.html
http://www.wizardworld.com/w91/starwars/starwars2.html
http://www.corona.bc.ca/films/details/bountyhunters.html
http://mp4radio.com/artists/9/pile_of_eggs_poe.html
http://master.mp3.com/artists/9/pile_of_eggs_poe.html
http://houston-press.com/1998/012998/mdance.html
http://www.bluemoon.net/~jkcioppa/sotkstarwars.html
http://www.fuzzy.nl/marsty/StarWars/SWCollecting/POTFWorld/potf_usa.htm
http://www.usmo.com/~starwars/figures.html
http://www.toptown.com/THEEDGE/BIKERSCOUT/news.htm
http://pack.d0gz.net/canines/Volcano/Efficiency.html
http://www.people.cornell.edu/pages/dlf14/Reviews/front.html
http://mail.mc2.nu/~secure/msg00975.html
http://www.geek-girl.com/bugtraq/1999_1/0167.html
http://www.filelibrary.com/Contents/Multi-Platform/104/76.html
http://frontpage.lightspeed.net/wolfman/ref.htm
http://www.beyond.swccgnews.com/articles/stlouisnewsletter2.html
http://www2.cybernex.net/~apagan/complete.html
http://www.figuresinc.com/c1.htm
http://www.wiseacres.com/skenner3.htm
http://csclub.stthomas.edu/~bugtraq/1999/msg00158.html
http://www.toy-palace.com/
http://www.episode1.de/
http://vgames.webpoint.com/vid/hamptonroads/games/0,1490,447,00.html
http://www.hoser.com/hotwire/q2ctfstats/sep1998/Outlands/Skill-top3.html
http://www.ronaldo.com/bounty
http://www.asylum.com/starwars
http://wso.williams.edu/~rfoxwell/starwars/SWPics.htm
http://www.galoob.com/SWMPL.html
http://user.mc.net/loechle/xvt.htm
http://holocron.simplenet.com/issues/hv3i3/tips.htm
http://www1.linkonline.net/fresh/disc.htm
http://www.playhere.com/nintendo64/shadowoftheempirefaq.htm
http://starwars.priv.pl/e1/corus/corus.htm
http://members.home.com/fresh/rephlex/roster.htm
http://www.primenet.com/~btn/atrivia.html
http://www.n64codemaster.com/shadows.htm
http://www.cis.ohio-state.edu/hypertext/faq/usenet/starwars/www/part3/faq.html
http://www.yakface.com/dlrproductions/pricing.html
http://www.n64cc.com/SOTEFAQ.HTM
http://www.solnet.co.uk/wonderland/toys3.html
http://www.sirius.com/~fonzilla/sw.htm
http://www.gamenexus.com/hints/n64/shadows64.sht
http://www.users.globalnet.co.uk/~sward
http://home.epix.net/~bgrose
http://www1.seeder.net.tw/bbs/game/980205.htm
http://force.stwing.upenn.edu/starwars/films/quizbook.txt
http://www.ionet.net/~theguy/reference.html
http://www.wwwfights.com/History/fett_pred.html
http://www.bit.net.au/pwebstats/days/hosts/foobar.190.total-hosts.html
http://www.webring.org/ringworld/ent/movies.html
http://www.phoaks.com/rec/arts/sf/starwars/misc/resources0.html
http://webring.net/ringworld/ent/movies.html
http://www.monolithtoys.com/classifieds.htm
http://www.praetor.com/gab/link.htm
http://www.ncf.carleton.ca/ip/sigs/futurist/star-wars/faq/movie
http://hjem.get2net.dk/star-wars
http://www.playhere.com/nintendo64/shadowoftheempirefaq.htm
http://www.videogamers.com/n64cp/sote.htm
http://starwars.priv.pl/e1/corus/corus.htm
http://members.home.com/fresh/rephlex/roster.htm
http://www.primenet.com/~btn/atrivia.html
http://www.n64codemaster.com/shadows.htm
http://www.alphalink.com.au/~bens/case.html
http://web.qx.net/red6/updates.htm
http://www.yakface.com/dlrproductions/faqs.html
http://www.everyicon.com/cheatcodes/pccheats/shadowsoftheempire.htm
http://www.solnet.co.uk/wonderland/toys3.html
http://www.moseisley.com/forum
http://www.sirius.com/~fonzilla/sw.htm
http://world.std.com/~scifinds/starwarslist.html
http://www2.msstate.edu/~jwf2
http://www.iserv.net/~tgd/cheats/r-z/star-wars-shadows.htm
http://www.gamenexus.com/hints/n64/shadows64.sht
http://www.users.globalnet.co.uk/~sward
http://home.epix.net/~bgrose
http://comunidades.infosel.com/starwars/referencia/glosarios/1759/\
http://www.n64cc.com/SOTEFAQ2.HTM
http://starwars.at/maxrebo
http://www1.seeder.net.tw/bbs/game/980205.htm
http://www.stwing.upenn.edu/starwars/films/quizbook.txt
http://www.fanatech.com/prebooks.html
http://www.ionet.net/~theguy/reference.html
http://www.wwwfights.com/History/fett_pred.html
http://www.bit.net.au/pwebstats/days/hosts/foobar.190.total-hosts.html
http://www.hyperreal.org/raves/mw/calendar/cal
http://www.phoaks.com/rec/arts/sf/starwars/misc/resources0.html
http://www.anth.ucsb.edu/feedback/visitors8.html
http://www.flash.net/~lkingc/INDEX1.HTM
http://stampo.com/log/lists/sites1097.html
http://www.auggy.mlnet.com/bt/Rank.html
http://www.erols.com/njesuele/script2.htm
http://www.bus.miami.edu/~jdavis/starwars/quotes.html
http://www.engr.orst.edu/~volzmi/starwars/empire.txt
http://stwing.upenn.edu/starwars/films/tesb.txt
http://directions.simplenet.com/hobbes/starwars/empire.html
http://www.hopwood.ac.uk/stats/aug96.htm
http://phantom.skywalk.com/pwebstats/weeks/hosts/phantom.skywalk.com.24.total-hosts.html
http://yomama.bus.miami.edu/~jdavis/StarWars/quotes.html
http://www-fib.upc.es/ESTADISTIQUES/Est9697/sites0797.html
http://xplor.csb.yale.edu/http-analyze/sites0698.html
http://www6.geocities.com/Hollywood/9990/facts2.html
http://www.cusd.claremont.edu/reports/oldlogs/Oct98.wwwstats.html
